### PR TITLE
Doc: update readme.md 

### PR DESCRIPTION
**Pull request by akshitarora921**
- The link on token-generation leading to `https://docs.100ms.live/javascript/v2/guides/token` is misleading the point of generating token, I myself wasted 1 hour.
- Update the env variable name to the field name in the website ie. Token endpoint, not Token generation endpoint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ Create a new file `.env` and copy the values from `example.env`
...
 ```
 
-### Get the token generation endpoint
+### Get the token endpoint
 
-Get your [token generation](https://docs.100ms.live/javascript/v2/guides/token) endpoint from the [Developer section of 100ms' Dashboard](https://dashboard.100ms.live/developer)
+Get your token endpoint from the [Developer section of 100ms' Dashboard](https://dashboard.100ms.live/developer)
 
-Update the `REACT_APP_TOKEN_GENERATION_ENDPOINT` in `.env` file with your token generation service endpoint (eg. `https://prod-in.100ms.live/hmsapi/example.app.100ms.live/`)
+Update the `REACT_APP_TOKEN_ENDPOINT` in `.env` file with the above token value (eg. `https://prod-in.100ms.live/hmsapi/example.app.100ms.live/`)
```
Comment (Deep-Codes) [inline README.md]: Why change from `REACT_APP_TOKEN_GENERATION_ENDPOINT` to `REACT_APP_TOKEN_ENDPOINT` ?

Comment (akshitarora921) [reply README.md]: At this link `https://dashboard.100ms.live/developer` its name is Token endpoint, so for consistency.

**Comment by akshitarora921** [comment]
@raviteja83 Please review.

**Comment by triptu** [comment]
Hi @akshitarora921, thanks for the PR. Renaming the env variable however be a significant breaking change for us. It'll require changes across places in our internal tooling as well as a breaking change for folks who are working with a fork of this repo(secrets etc. will need to be changed for env variables to make the deployment work). Do you think adding an image in the readme to better explain the meaning of the endpoint would have helped as well?

**Comment by akshitarora921** [comment]
@triptu thanks for the feedback, I will add the image to explain which value we are referring to.

**Comment by akshitarora921** [comment]
@triptu please review.

**Review by triptu — APPROVED** [review]

**Comment by triptu** [comment]
@akshitarora921 Merged. Thanks a lot!

**Pull request merged**